Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM debian:stretch
+# Install needed packages
+RUN apt-get update && apt-get -y install cmake make build-essential
```

Comment:
build-essential already depends on make (see (Debian doc)[https://packages.debian.org/stretch/build-essential]).

The CMake in stretch is 3.7.2, I'm not sure if that is enough to build runit (probably missing support for object libraries). If you have tested this and it works, please comment on #11 so that the correct version can be requested there.